test/Makefile

```diff
@@ -17,25 +16,21 @@ GUEST_BACKEND = 1
 
 define test_host
 	@cd host/ && \
-	$(py) runTests.py -m $(1) -s $(SECVAR_TOOL) && \
-	$(py) runSvcGenerateTests.py -m $(1) -o $(OPENSSL) -g $(GNUTLS) -s $(SECVAR_TOOL)
+	$(py) runTests.py -s $(SECVAR_TOOL) && \
+	$(py) runSvcGenerateTests.py -o $(OPENSSL) -g $(GNUTLS) -s $(SECVAR_TOOL)
 endef
 
 define test_guest
 	@cd guest/ && \
-	$(py) guest_tests.py -m $(1) -s $(SECVAR_TOOL)
+	$(py) guest_tests.py -s $(SECVAR_TOOL)
```

Comment:
please remove the `-m` arguments from guest and host python tests. Additionally any `MEMCHECK` and`valgrind` references can be removed as well.

Reply:
Slated for a later rework of the tests. Most of the "helper" code for the tests can be factored out into shared classes, will tackle that soon.